src/styles/main.scss

```diff
@@ -0,0 +1,51 @@
+$days: 31;
+$month-length: 28, 29, 30, 31;
+$start-day: mon, thu, wed, thu, fri, sat, sun;
```

Comment:
It's better to use lists (parentheses) for storing multiple values in SCSS variables. For example, use `$month-length: (28, 29, 30, 31);` instead of `$month-length: 28, 29, 30, 31;`. This will help to avoid confusion with comma-separated lists of values for CSS properties.